client/interactions.lua

```diff
@@ -354,10 +352,9 @@ RegisterNetEvent('police:client:GetKidnappedTarget', function(playerId)
     end)
 end)
 
-RegisterNetEvent('police:client:GetKidnappedDragger', function(playerId)
-    QBCore.Functions.GetPlayerData(function(PlayerData)
+RegisterNetEvent('police:client:GetKidnappedDragger', function()
+    QBCore.Functions.GetPlayerData(function(_)
```

Comment:
If this is not used, there is jo point in keeping this callback, you can remove the QBCore.Functions.GetPlayerData function call and remove the function parameter but place the code back which was inside the callback after you're done